client/lib/track-form/index.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { union } from 'lodash';
+
+export const trackForm = WrappedComponent => {
```

Comment:
no need for this `{` and the `return` below. because `class` isn't a real class in JavaScript they can be returned as `class expression`

```js
export const trackForm = WrappedComponent => class extends Component {

};
```